src/components/ExploreCarousel/index.js

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import { H3, H2, StyledLink } from "../../styles/styles";
+import {
+  FlexContainer,
+  responsiveWatch,
+  responsiveRead,
+  FlexRow,
+  Work,
+  LeftIcon,
+  RightIcon,
+  StyledP,
+  StyledImage,
+  StyledArticleImage,
+  ItemContainer,
+} from "./styled";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+import { PrismicRichText } from "@prismicio/react";
+
+const ExploreCarousel = ({ section }) => {
+  const {
+    data: { title, works },
+    uid: sectionuid,
+  } = section;
+
+  const CustomLeftArrow = ({ onClick }) => (
+    <LeftIcon onClick={() => onClick()} />
+  );
+  const CustomRightArrow = ({ onClick }) => {
+    return <RightIcon onClick={() => onClick()} />;
+  };
+
+  const SingleCarousel = () => (
+    <Carousel
+      responsive={responsiveWatch}
+      customRightArrow={<CustomRightArrow />}
+      customLeftArrow={<CustomLeftArrow />}
+      infinite={true}
```

Comment:
any way you can get this to not play automatically after you click the arrows? maybe `autoPlay={false}`? if it doesn't work nbd

Reply:
ty that worked to turn off autoplay